### ISSUE TITLE
Substitution does not register when applied to a message with attached image

### ISSUE THREAD

**Issue by serid** [GoldsteinE/perlsub#1]
Replying to a message that consists of an image with a caption does not trigger a reaction from the bot. I am presenting an example interaction where the bot is notably silent despite the substitution command being visible in chat log. Expected behavior in response is up for discussion. I see two options:
1. When issued a command to substitute a message with a caption and an image (album of images) attached, the bot replies with a message consisting of same image (album of images) and a substituted caption.
2. When issued a command to substitute a message with a caption and an image (album of images) attached, the bot replies with a substituted caption, omitting the image since this type of media can not be processed by the Perl regex engine.
![image](https://github.com/GoldsteinE/perlsub/assets/30107608/04979da7-37d0-4c41-b95f-cb8fa3bdbd3f)

**Comment by GoldsteinE** [comment]
Yeah, that’s a known problem. I’m kind of against option 1, because it would look spammy, especially for big albums, option 2 seems reasonable. I don’t have resources to implement this myself right now, but I’ll accept a PR implementing option 2.
